### PR TITLE
Add rebaseRules for avi-secrets to retain user modifications to avi c…

### PR DESCRIPTION
**Pull request by sbhatp**
Add rebaseRules for avi-secrets to retain user modifications to avi credentials

Signed-off-by: Sharath Bhat <sharathb@vmware.com>

## Details for the Release Notes (PLEASE PROVIDE)

```release-note

```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Comment by stmcginnis** [comment]
For future PRs, please create a branch in your own fork, then create the PR from there.

**Review by HanFa — APPROVED** [review]
LGTM

File: addons/packages/ako-operator/1.6.0/bundle/config/kapp-config.yaml [hunk 1]
```diff
@@ -23,3 +23,15 @@ data:
       sources: [existing, new]
       resourceMatchers:
       - apiVersionKindMatcher: {apiVersion: networking.tkg.tanzu.vmware.com/v1alpha1, kind: AKODeploymentConfig}
+    - paths:
+      - [data, username]
+      - [data, password]
+      type: copy
+      sources: [existing, new]
+      resourceMatchers:
+      - kindNamespaceNameMatcher: {kind: Secret, namespace: tkg-system-networking, name: avi-controller-credentials}
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.6.0/bundle/config/kapp-config.yaml:32]: This is the default name, you can use YTT overlay to get the input from `ako-operator` add-on secret

File: addons/packages/ako-operator/1.6.0/bundle/config/kapp-config.yaml [hunk 2]
```diff
@@ -23,3 +23,15 @@ data:
...
+    - path: [data, certificateAuthorityData]
+      type: copy
+      sources: [existing, new]
+      resourceMatchers:
+      - kindNamespaceNameMatcher: {kind: Secret, namespace: tkg-system-networking, name: avi-controller-ca}
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.6.0/bundle/config/kapp-config.yaml:37]: Same here, it should be the input from `ako-operator` 's addon secret.

**Pull request merged**